src/v2/Components/v2/ArtworkFilter/ArtworkFilters/TimePeriodFilter.tsx

```diff
@@ -70,4 +70,5 @@ const allowedPeriods = [
   "Late 19th Century",
   "Mid 19th Century",
   "Early 19th Century",
+  "18th Century & Earlier",
```

Comment:
Nice 🎉 

I also spot checked some results of the data migration. Overall looks pretty good, except for some of the quirky input you mentioned:

![Screen Shot 2020-09-21 at 11 50 13 AM](https://user-images.githubusercontent.com/140521/93790313-e3c8ab00-fc00-11ea-9625-c10f7b870ef3.png)

![Screen Shot 2020-09-21 at 11 49 41 AM](https://user-images.githubusercontent.com/140521/93790315-e3c8ab00-fc00-11ea-8b15-db4bbe67e499.png)

![Screen Shot 2020-09-21 at 11 46 48 AM](https://user-images.githubusercontent.com/140521/93790316-e3c8ab00-fc00-11ea-8f6c-b98b42d37c30.png)

But fair to say this is a big improvement over the incorrectly parsed year ranges.